docs/source/about/advanced_usage.rst

```diff
@@ -466,7 +466,7 @@ audio_sink
    **Windows**
       .. code-block:: text
 
-         audio_sink = {0.0.0.00000000}.{FD47D9CC-4218-4135-9CE2-0C195C87405B}
+         audio_sink = Speakers (High Definition Audio Device)
```

Comment:
Should we have two examples? One with the old ID method, in the event a user has multiple of the same audio device when going by name (e.g. identical monitors)

Reply:
Identical monitors will actually have unique names. I really don't want to encourage use of the old IDs because they tend to break during normal things like GPU/audio driver updates or connecting a new monitor.

![image](https://user-images.githubusercontent.com/2695644/236372892-734e0406-229e-4305-820b-299c3f6796e4.png)

Reply:
That's interesting. I cannot test right now, but I was pretty sure I wasn't getting unique names with my old 3 HP monitor setup.

Reply:
My initial test was on a Win11 PC with an Intel GPU. I tested on a Win10 PC with an AMD GPU and saw the same results with those `-2` and `-3` strings appended to make the name unique. However, this particular system has a duplicate SPDIF output device for some reason, so it does seem that duplication is possible in general even if it doesn't seem to happen for displays.

I'll add a note to the docs and the UI to prefer names but use IDs if the names are duplicated.